source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md

```diff
@@ -0,0 +1,115 @@
+---
+title: App Development on OpenShift using ADB
+author: praveenkumar
+date: 2016-04-27 22:50:00 UTC
+tags: openshift, origin, ADB
+published: true
+comments: true
+---
+
+[The Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) is a prepackaged development environment filled production-grade pre-configured tools which also include [OpenShift origin](https://github.com/openshift/origin). Using ADB app-developers can easily start building and developing their application on OpenShift platform.
+
+READMORE
+
+In this blogpost we are going to learn step by step to create an application on OpenShift platform and deploy. Before we proceed any further I highly recommend you to go through [prerequisites guide](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst). ADB provide a [custom Vagranfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile) to setup and provision OpenShift platform.
+
```

Comment:
IMO we should start the steps from git clone ADB repo , then going to the directory where the Vagrantfile for OpenShift exists and then doing a `vagrant up`.